diff -r 0d86faf4b05c config.def.h
--- a/config.def.h	Sun Jul 25 09:58:25 2010 +0100
+++ b/config.def.h	Sun Jul 25 16:32:38 2010 +0200
@@ -5,4 +5,5 @@
 static const unsigned int borderpx  = 1;        /* border pixel of windows */
+static const Bool viewontag         = True;     /* Switch view on tag switch */
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
diff -r 0d86faf4b05c dwm.c
--- a/dwm.c	Sun Jul 25 09:58:25 2010 +0100
+++ b/dwm.c	Sun Jul 25 16:32:38 2010 +0200
@@ -1613,6 +1613,8 @@
 	if(selmon->sel && arg->ui & TAGMASK) {
 		selmon->sel->tags = arg->ui & TAGMASK;
 		arrange(selmon);
+		if(viewontag)
+			view(arg);
 	}
 }
 
